prov/efa/test/efa_unit_test_cq.c

```diff
@@ -74,14 +74,13 @@ void test_ibv_cq_ex_read_empty_cq(struct efa_resource **state)
  * then user should call fi_cq_readerr() to get an error CQ entry that contain error code.
  *
  * @param[in]  state            struct efa_resource that is managed by the framework
- * @param[in]  ep_type          Endpoint type, can be FI_EP_DGRAM or FI_EP_RDM
  * @param[in]  local_host_id    Local(sender) host id
  * @param[in]  peer_host_id     Peer(receiver) host id
  * @param[in]  vendor_error     Vendor error returned by ibv_read_vendor_err
  * @param[in]  err_prefix       Expected error message prefix from fi_cq_strerror. For RDM endpoint the error
  *                              might contain conditional information after the prefix.
  */
-static void test_cq_read_bad_send_status(struct efa_resource *resource, enum fi_ep_type ep_type,
+static void test_cq_read_bad_send_status(struct efa_resource *resource,
```

Comment:
I think removal of FI_EP_DGRAM from these tests should come first as a separate commit.

I see why the rest of these changes happen during this commit.

Reply:
Did that in a separate commit

Reply:
thanks!